templates/hooks/user/sdk_create_post_set_output.go.tpl

```diff
@@ -1,3 +0,0 @@
-    if err := rm.syncPolicies(ctx, &resource{ko}); err != nil {
-        return nil, err
-    }
```

Comment:
I removed, `sdk_create_post_set_output` when they only sync policies or tags. This will be quickly handled by the update code path.